test/test_common/utility.h

```diff
@@ -556,9 +557,15 @@ class TestUtility {
 
   template <class MessageType>
   static void loadFromYamlAndValidate(const std::string& yaml, MessageType& message,
-                                      bool preserve_original_type = false) {
-    MessageUtil::loadFromYamlAndValidate(yaml, message,
-                                         ProtobufMessage::getStrictValidationVisitor());
+                                      bool preserve_original_type = false,
+                                      bool avoid_boosting = true) {
+    if (avoid_boosting) {
```

Comment:
nit: I think this new code block can become: `MessageUtil::loadFromYamlAndValidate(yaml, message, ProtobufMessage::getStrictValidationVisitor(), !avoid_boosting);`